### ISSUE TITLE
A few build issues

### ISSUE THREAD

**Issue by JEphron** [aminya/d-tree-sitter#4]
I encountered some issues getting this library installed and building reliably. They may have been specific to my system, but I'll document them here for posterity and in case anyone else runs into similar snags. 

I needed to remove the `--no-sys-headers` flag to get it to find `stddef.h`

in `/script/d_tree_sitter.d`
```diff
-  const headers = " --ignore-system-paths --no-sys-headers " ~ prebuiltHeaders;
+  const headers = " --ignore-system-paths " ~ prebuiltHeaders;
```
and in `dub.sdl` I needed to adjust my `lflags`
```diff
-lflags "./src/build/libtree_sitter.a"
+lflags "$PACKAGE_DIR/src/d_tree_sitter/build/libtree_sitter.a"
```

**Comment by aminya** [comment]
Are you on MacOS? Because I had trouble building the project on Mac. We can try your changes inside CI, and merge them.

**Comment by JEphron** [comment]
This was on (Arch) Linux. I'm sure I'll end up debugging this on MacOS at some point though.

**Comment by aminya** [comment]
Build on MacOS is now added to CI. You might try on Arch Linux and let me know.

**Comment by aminya** [comment]
This fix is applied in #8
